### PR TITLE
Feature/document

### PR DESCRIPTION
**Pull request by ndkhanh-axonivy**

### CODE REVIEW

**Comment by ndkhanh-axonivy** [comment]
Hi @andreasbalsiger I created this pull request for the documentation step.
Could you help review this README.md file, it describes the purposes and functionality for the UPS connector.
Thanks!

**Comment by ndkhanh-axonivy** [comment]
Hi Andreas,

Thank you for reviewing the pull request. I apologize for the confusion and any inconvenience caused by closing it before your edits were fully incorporated.

I recently realized that the original pull request wasn't based on the latest history due to another pull request merging into the master branch first. Unfortunately, directly merging onto older commits can lead to conflicts and make collaboration more difficult.

To address this, I've recreated the pull request with the updated code, including your reviewed edits from the previous version. You can find the new pull request here: .

I understand this creates some extra work for you, and I sincerely apologize for not communicating the issue before closing the first pull request. I'm still learning the best practices for collaborating on code revisions, and I appreciate your patience and understanding.

Please feel free to review the new pull request at your convenience and let me know if you have any further questions or edits.

Thank you again for your time and feedback.

Best regards,
Khanh


________________________________

**Comment by andreasbalsiger** [comment]
There is a missing link in this email below.


________________________________
Von: Khanh Nguyen ***@***.***>
Gesendet: Donnerstag, 22. Februar 2024 12:27
An: axonivy-market/ups-connector ***@***.***>
Cc: Andreas Balsiger ***@***.***>; Mention ***@***.***>
Betreff: Re: [axonivy-market/ups-connector] Feature/document (PR #5)


Hi Andreas,

Thank you for reviewing the pull request. I apologize for the confusion and any inconvenience caused by closing it before your edits were fully incorporated.

I recently realized that the original pull request wasn't based on the latest history due to another pull request merging into the master branch first. Unfortunately, directly merging onto older commits can lead to conflicts and make collaboration more difficult.

To address this, I've recreated the pull request with the updated code, including your reviewed edits from the previous version. You can find the new pull request here: .

I understand this creates some extra work for you, and I sincerely apologize for not communicating the issue before closing the first pull request. I'm still learning the best practices for collaborating on code revisions, and I appreciate your patience and understanding.

Please feel free to review the new pull request at your convenience and let me know if you have any further questions or edits.

Thank you again for your time and feedback.

Best regards,
Khanh


________________________________

**Comment by ndkhanh-axonivy** [comment]
Hi Andreas,

The link is https://github.com/axonivy-market/ups-connector/pull/5.

Best regard,
Khanh

________________________________

**Comment by andreasbalsiger** [comment]
Done!
________________________________
Von: Khanh Nguyen ***@***.***>
Gesendet: Donnerstag, 22. Februar 2024 15:20
An: axonivy-market/ups-connector ***@***.***>
Cc: Andreas Balsiger ***@***.***>; Mention ***@***.***>
Betreff: Re: [axonivy-market/ups-connector] Feature/document (PR #5)


Hi Andreas,

The link is https://github.com/axonivy-market/ups-connector/pull/5.

Best regard,
Khanh

________________________________

**Comment by ndkhanh-axonivy** [comment]
Good morning Andreas,

Thanks for reviewing PR! I didn't see any commits pushed yet. Is there anything you need from me to finalize the changes?

Best regards,
Khanh

**Review by andreasbalsiger — COMMENTED** [review]
My final edits for you to change.

**Comment by ndkhanh-axonivy** [comment]
Hi Andreas,

Thank you so much for your feedbacks, I also pushed the updated code. Could you please check it again!

Best regards,
Khanh

File: ups-connector-product/README.md [hunk 1]
```diff
@@ -1,74 +1,73 @@
...
-We hope you enjoy and we look forward to your contributions!
-
-## Set up
-
-1. Go to https://developer.ups.com, login or create a new UPS account. 
-2. Create an Application on UPS 
-3. After creating your application, the **Client ID** and **Client Secret** are generated and can be used to obtain an access token to authorize your API requests
-4. Configure the following variables in your project: 
-![Alt text](image-11.png)
+The Axon Ivy' [UPS connector](https://developer.ups.com/catalog) enables users to integrate UPS services seamlessly into any business process. This connector:
```
Comment (andreasbalsiger) [inline ups-connector-product/README.md:4]: "The Axon Ivy" and not "The Axon Ivy' "

Comment (andreasbalsiger) [reply ups-connector-product/README.md:4]: can you make this change and then you can comit and merge it ok?

Comment (ndkhanh-axonivy) [reply ups-connector-product/README.md:4]: Thank you so much Andreas, have a nice day!

**Pull request merged**